python/tvm/relay/frontend/tensorflow.py

```diff
@@ -34,6 +34,17 @@
 
 __all__ = ['from_tensorflow']
 
+def _infer_value(input_val, params):
```

Comment:
We could add additional check to make sure all inputs to infer is available in params.
A utility similar to list_input_names() in nnvm.

Not sure if we have similar API in relay.
cc @zhiics @jroesch @tqchen

Reply:
Do you think something like this function could help?
https://github.com/dmlc/tvm/blob/master/python/tvm/relay/backend/interpreter.py#L136

Reply:
I doubt it fits here.
What we need here is list of Var nodes current Expr is dependent on recursively.

Reply:
Can you just take free_vars if it is an expr? Var has name_hint.

Reply:
free_vars should work.

@jwfromm can you cross check free_vars with params and add an assert ?

Reply:
Added in the latest push.